draft/spec/index.html

```diff
@@ -238,14 +238,15 @@ <h2>Terminology</h2>
             <dfn>Logical State</dfn>:
         </dt>
         <dd>
-            A grouping of logical file paths reflecting the state of the content for a given version.
+            A grouping of logical file paths with particular content reflecting the state of the object content for
```

Comment:
'content' ... 'content' is awkward. Not sure what the 'particular content' here refers to?

Reply:
The previous text for this sentence seems more clear to me. Is there a particular essence that this update is trying to clarify?

Reply:
Yes, the old sentence is wrong in my opinion because it says the state is only the file paths. The state is actually the pointers to the (particular) content and their paths. I agree that the particular didn't work, proved in part because it didn't make sense to either of you. How about:

> A grouping of logical file paths tied to their corresponding bitstreams that reflect the state of the object content for a given version.